packages/app/src/update/helpers/version-formatter.helper.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export const versionFormatter = (version: string): string => {
+  const [releaseVersion, suffix] = version.split("-")
+
+  if (!suffix) {
+    return version
+  }
+
+  const releaseSuffixArray = suffix.split(".")
+
+  if (releaseSuffixArray.length < 4) {
+    return version
+  }
+
+  // AUTO DISABLED - fix me if you like :)
+  // eslint-disable-next-line prefer-const
+  let [name, year, month, day] = releaseSuffixArray
+  month = month?.replace(/0/g, "")
+  day = day?.replace(/0/g, "")
```

Comment:
Remove the lint rule or refactor a code 😉 
```suggestion
  const [name, year, m, d] = releaseSuffixArray
  const month = m?.replace(/0/g, "")
  const day = d?.replace(/0/g, "")
```

Reply:
https://github.com/mudita/mudita-center/pull/979/commits/ee21f435f9bab9e418666e907b7c3198820fde9f